### PR TITLE
Private Events 

### PR DESCRIPTION
**Pull request by gdumani**
- Sign up to the site
- Sign in to the site
- Create events for others to attend
- Check own events, current and past ones
- Check available events and add them to attending events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,82 @@
...
+* Ruby 3.0
+* Rais 6
+* SQL lite
 
-* Ruby version
+## Getting started
 
-* System dependencies
+To clone and run the application locally, use the following steps:
+- Clone the repository from GitHub, using the `git clone` command.
```
Comment (BrianSammit) [inline README.md]: Where do I need to run this command. 
Write the whole command that they need to run it correctly. 
`git clone https://github.com/gdumani/private-events.git`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,82 @@
...
+- Create the new application and switch to its folder `$cd folder name`.
```
Comment (BrianSammit) [inline README.md]: The people don't need to create the new app. It's already created with the `git clone`

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGES REQUIRED :red_circle: :stop_sign: 
--------------
Hello @gdumani @Takhmina175 :wave: You did an amazing job so far. 
TO HIGHLIGHT: 
- Your project is working really nicely. 
- You have a well structured files. 
- Your repository is professional.
- Your project is professional.
However, there are a few changes to make to complete this project. :hammer_and_wrench: see them below :eye: :point_down: 
----------------
- [x] You don't have an event's show page that displays the list of attendants. Please create a show page for a single event that shows a list of attendees. 

events_attendees_requirement

- [x] Please improve your commit messages. Remember the formating [rules](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules). You must use the imperative for the commit messages. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read about how to changes a commit message. You don't need to changes the commit messages this time but keep that formatting rules in mind for the next commits. 
- [x] You are not using `index.html.erb` in users folder inside your views. Please remove this file. 
- [x] Improve the steps In your readme file on Getting started. The steps are correct but write them thinking on someone that doesn't know anything about programming. See my inline comments. 
- [x] Write the steps on how to run the tests. You have steps about how to install RSpec. but you don't have steps on how to run the tests. 
- [x] [OPTINAL] Put a screenshot on your Readme file.
------------------
Make the required changes and send it again for another review :muscle: 
Reachme on SALCK if you have any questions. 
@ Brian Sammit Cruz Rodriguez
HAPPY CODING!! :smile: 

MB5W4D

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :red_circle: 

### Hello @gdumani  and @Takhmina175, thank you for making almost all the required changes from the previous TSE, your project is almost complete, but please also implement the following requirement asked previously:
- [x] _**Please improve your commit messages. Remember the formating [rules](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules). You must use the imperative for the commit messages. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read about how to changes a commit message. You don't need to changes the commit messages this time but keep that formatting rules in mind for the next commits.**_

_Please implement all the above mentioned requirements and ask for another code review whenver you are ready_

### **If you have any question related to the requirements please add a comment here and ping me on slack @GzimAsani**

HAPPY CODING :computer: :desktop_computer:

**Comment by gdumani** [comment]
That requirement is already implemented in accordance to the reviewer's specification. There was no request to change previous commits. 

> Please improve your commit messages. Remember the formating rules. You must use the imperative for the commit messages. Here you can read about how to changes a commit message. _**You don't need to changes the commit messages this time but keep that formatting rules in mind for the next commits.**_

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED :green_circle: :clap: 
--------------
Hello, @Takhmina175 @gdumani  You make an amazing job working on the required changes. :muscle: 
---------
Now you can merge this branch :+1: :100: 
HAPPY CODING!!!

**Pull request merged**